src/DiffEqBiological.jl

```diff
@@ -5,6 +5,8 @@ module DiffEqBiological
 using Compat, DataStructures, MacroTools, Parameters, Reexport, SparseArrays, SymEngine
 using DiffEqBase, DiffEqJump
 @reexport using DiffEqBase, DiffEqJump
+using HomotopyContinuation, DynamicPolynomials, LinearAlgebra, RecipesBase
+@reexport using DynamicPolynomials
```

Comment:
why reexport?

Reply:
It has to do with the polyvars being created in the scope in which the macro is called, and some irritating issues that appeared were polyvars could have same names, but be different.

However, after your comment I had a look at this again. Due to the revised approach were I do a lot of stuff in the EquilibrateContent constructor I was able to move were the polyvars were created and could remove this reexport.